packages/mongo-menu/src/MongoMenu.tsx

```diff
@@ -168,14 +168,16 @@ const menuItems = [
   },
 ] as const;
 
-export const Product = {
+const Product = {
   Atlas: 'atlas',
   University: 'university',
   Support: 'support',
 } as const;
 
 type Product = typeof Product[keyof typeof Product];
 
+export { Product };
```

Comment:
Likewise, this fixes a small typescript error in the stories for MongoMenu: 
(1) the array of products was typed erroneously as `Array<Size>` instead of `Array<Product>` in the stories
(2) we exported Product as data rather than as a type (I checked this against several other packages to ensure exporting the type was the intent)